Assets/Scripts/Game/RegionSelector.cs

```diff
@@ -23,15 +28,25 @@ private void Awake()
 
 		private void OnMouseDown()
 		{
+			if (GameUIController.Instance.IsPointerOverUI(Input.mousePosition)) return;
```

Comment:
Bedre om vi bruker det gamle ui-systemet for dette. Det nye er ikke laget for å være i world space, så det gjør det en del vanskeligere. Med den gamle måten å gjøre det på kan man bare skrive EventSystem.current.IsMouseOverGameObject();